perftest/wallet.go

```diff
@@ -103,9 +103,11 @@ func (w walletWrapper) NewMarket(offset int, user UserDetails) error {
 			"enactmentTimestamp": w.SecondsFromNowInSecs(30),
 			"newMarket": map[string]interface{}{
 				"changes": map[string]interface{}{
-					"lpPriceRange":          "10",
-					"decimalPlaces":         "5",
-					"positionDecimalPlaces": "5",
+					"linearSlippageFactor":    "1",
```

Comment:
maybe it doesn't matter but why not use proper recommended values?

Reply:
I have updated to use the default 0.001 and 0.0 values, thanks for the heads up!